src/pages/ServerError.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import DisstressedTubey from '../common/images/distressed-tubey.svg';
+import './NoMatch/NoMatch.scss';
+
+let ServerError = () => (
+  <div className="error-page">
+    <Helmet>
+      <title>We’re a little overwhelmed at the moment - refine.bio</title>
+    </Helmet>
+    <div className="error-page__text">
+      <h1>We’re a little overwhelmed at the moment.</h1>
+
+      <p>
+        We apologize for the inconvenience. We are working hard to restore
+        normal service.
+      </p>
+      <p>
+        Follow us on twitter (
+        <a
+          href="https://twitter.com/CancerDataLab"
+          target="_blank"
+          rel="nofollow noopener noreferrer"
+          className="link"
+        >
+          @CancerDataLab
+        </a>
+        ) for updates
+      </p>
+    </div>
```

Comment:
This page is almost identical to the one on `src/pages/Search/index.js`. Is it possible create a directive to avoid the duplication? (Feel free to add a new issue to address it later, if you don't have time now.)